can-connect.md

```diff
@@ -15,11 +15,11 @@ project from fully modularized behaviors (i.e. plugins).
 
 Iterate through passed behaviors and assemble them into a connection.
 
-```js
-var connect = require("can-connect");
-var dataUrl = require("can-connect/data/url/");
-var constructor = require("can-connect/constructor/");
-var todosConnection = connect([dataUrl, constructor],{
+```javascript
```

Comment:
Why did this change from just `js`?

Reply:
consistency. no particular reason

Reply:
This is fine, but note that even [some of our own tools don’t support javascript](https://github.com/canjs/can-migrate/issues/38).

Reply:
I don't really mind either way. They use js now.